IlmBase/IlmThread/IlmThreadPool.cpp

```diff
@@ -504,14 +508,14 @@ TaskGroup::Data::~Data ()
 
 
 void
-TaskGroup::Data::addTask () 
+TaskGroup::Data::addTask ()
 {
     //
     // in c++11, we use an atomic to protect numPending to avoid the
     // extra lock but for c++98, to add the ability for custom thread
     // pool we add the lock here
     //
-#ifdef ILMBASE_FORCE_CXX03
+#if ILMBASE_FORCE_CXX03
```

Comment:
We didn't intend for this to be always defined, which #if requires. Is there a reason why you might wish to force it to be zero or one?